### PR TITLE
Added deps list to include must install packages

### PR DESCRIPTION
**Pull request by abdhaleegit**
few packages needs to be checked and installed like patch
 make, with out this packages skip the tests.

Signed-off-by: Abdul Haleem <abdhalee@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by PraveenPenguin** [comment]
@abdhaleegit as i requested in https://github.com/avocado-framework-tests/avocado-misc-tests/issues/343 issue can you please look on repository as multiple test having same issue , this is really helpful

File: generic/interbench.py [hunk 1]
```diff
@@ -44,9 +44,11 @@ def setUp(self):
...
         '''
         sm_manager = SoftwareManager()
-        if (not sm_manager.check_installed("gcc") and not
-                sm_manager.install("gcc")):
-            self.error("Gcc is needed for the test to be run")
+        deps = ['gcc', 'patch', 'make']
+        for package in deps
+            if (not sm_manager.check_installed(package) and not
+                    sm_manager.install(package)):
+                self.error("%s is needed for the test to be run" % pacakge)
```
Comment (harish-24) [inline generic/interbench.py]: Can you change to self.skip here?

File: perf/lmbench.py [hunk 2]
```diff
@@ -47,8 +47,10 @@ def setUp(self):
         temp_file = self.params.get('temp_file', default=None)
         self.tmpdir = tempfile.mkdtemp(prefix='avocado_' + __name__)
         smm = SoftwareManager()
-        if not smm.check_installed("gcc") and not smm.install("gcc"):
-            self.error("Gcc is needed for the test to be run")
+        deps = ['gcc', 'patch', 'make']
+        for package in deps:
+            if not smm.check_installed(package) and not smm.install(package):
+                self.error("%s is needed for the test to be run" % package)
```
Comment (harish-24) [inline perf/lmbench.py]: here as well.

**Comment by PraveenPenguin** [comment]
@abdhaleegit please rebase with Pep8 compliance it will make happy Travis

**Comment by PraveenPenguin** [comment]
@abdhaleegit any update here ?

File: generic/interbench.py (hunk 1 above)
Comment (abdhaleegit) [inline generic/interbench.py]: self.skip is getting deprecated soon, shall I use self.cancel ?

**Pull request closed without merging**